modules/drivers/presto/src/metabase/driver/presto.clj

```diff
@@ -138,14 +144,15 @@
                 (if id
                   ;; If we have a query id, we can cancel the query
                   (try
-                    (http/delete (details->uri details-with-tunnel (str "/v1/query/" id))
-                                 (details->request details-with-tunnel))
+                    (let [tunneledUri (details->uri details-with-tunnel (str "/v1/query/" id))]
+                      (let [adjustedUri (create-cancel-url tunneledUri (get details :host) (get details :port) infoUri)]
```

Comment:
You can and should use a single `let` with multiple bindings instead of multiple lets 

```clj
(let [tunneled-uri (details->uri details-with-tunnel (str "/v1/query/" id))]
      adjusted-uri (create-cancel-url tunneledUri (get details :host) (get details :port) infoUri)]
   ...)
```